lalrpop/src/normalize/tyinfer/mod.rs

```diff
@@ -273,7 +273,8 @@ impl<'grammar> TypeInferencer<'grammar> {
             SymbolKind::Name(_, ref s) => self.symbol_type(&s.kind),
 
             SymbolKind::Repeat(..) | SymbolKind::Expr(..) | SymbolKind::Macro(..) |
-            SymbolKind::AmbiguousId(..) | SymbolKind::Lookahead | SymbolKind::Lookbehind => {
+            SymbolKind::AmbiguousId(..) | SymbolKind::Lookahead | SymbolKind::Lookbehind |
+            SymbolKind::Error => {
```

Comment:
Oops, that is a remnant from when I started and just needed something compiling after adding the `Error` variant.